patches/0004-Add-OpenSSL-crypto-backend.patch

```diff
@@ -199,7 +199,7 @@ index 00000000000000..69af0ffe2fcf80
 +// Use of this source code is governed by a BSD-style
 +// license that can be found in the LICENSE file.
 +
-+//go:build goexperiment.opensslcrypto && linux && cgo
++//go:build goexperiment.opensslcrypto && darwin && cgo
```

Comment:
Missing `linux`, and the filename also needs to change (I think simply calling it `openssl.go` is good) because currently it's impossible for this file to be included: https://pkg.go.dev/cmd/go#hdr-Build_constraints

> If a file's name, after stripping the extension and a possible _test suffix, matches [a suffix pattern] where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

(This causes `undefined: Enabled` because it's throwing off the other bits of the logic that assume the filename doesn't conflict with the `//go:build` line.)